FinSearchUnified/Tests/Bundle/SearchBundleFindologic/QueryBuilderFactoryTest.php

```diff
@@ -278,6 +290,22 @@ public function testCreateProductQueryWithOffsetAndLimit($offset, $expectedOffse
         );
     }
 
+    /**
+     * @dataProvider isSearchPageDataProvider
+     *
+     * @param bool $isSearchPage
+     * @param string $expectedInstance
+     *
+     * @throws Exception
+     */
+    public function testTypeOfQueryBuilder($isSearchPage, $expectedInstance)
+    {
+        Shopware()->Session()->offsetSet('isSearchPage', $isSearchPage);
+
+        $builder = $this->factory->createQueryBuilder();
+        $this->assertInstanceOf($expectedInstance, $builder, sprintf('Expected %s to be returned', $expectedInstance));
```

Comment:
The third parameter won't be necessary since `assertInstanceOf` will already give you a similar message.